### PR TITLE
Adding new methods for storage and tests

### PR DESCRIPTION
**Pull request by Benkdel**
Please help me reviewing this pull request

### CODE REVIEW

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,22 @@ def reload(self):
...
+        target = None
+        objs = self.__session.query(classes[cls.__name__]).all()
+        for obj in objs:
+            if obj.id == id:
+                target = obj
+        return target
+
+    def count(self, cls=None):
+        """ count objects in storage """
+        count= 0
```
Comment (CarlosGBentinDN) [inline models/engine/db_storage.py:89]: pycodestyle: variable and sign should have a space between them

**Review by CarlosGBentinDN — CHANGES_REQUESTED** [review]
Check comment in file storage, line 80 ichs

also, should change except with finally,

File: models/engine/db_storage.py (hunk 1 above)
Comment (Benkdel) [reply models/engine/db_storage.py:89]: will do

**Review by CarlosGBentinDN — APPROVED** [review]
its seems all ok now

File: models/engine/db_storage.py (hunk 1 above)
Comment (Benkdel) [reply models/engine/db_storage.py]: aaaa

**Pull request merged**